### PR TITLE
add a cleanup method for easy

### PR DESCRIPTION
**Pull request by natejgreene**
@ChrisArcand can you look at this one?   we are going to be explicitly freeing the easy in typhoeus

### CODE REVIEW

**Comment by chrisarcand** [comment]
Sure, but I'll need some time to dig in.

**Comment by natejgreene** [comment]
No need just need a CR. The QA will come as part of the typhoeus change

File: lib/ethon/easy/operations.rb [hunk 1]
```diff
@@ -26,6 +26,16 @@ def perform
         @return_code
       end
 
+      # Clean up the easy.
+      #
+      # @example Perform the request.
```
Comment (chrisarcand) [inline lib/ethon/easy/operations.rb]: Lies ;)

Comment (chrisarcand) [inline lib/ethon/easy/operations.rb]: Fix the description

**Pull request merged**